index.html

```diff
@@ -56,6 +56,156 @@ <h1>
+                        </ul>
+                        <button type="button" class="btn">See Project</button>
+                    </div>
+                </div>
+                <div class="work-item">
+                    <div class="work-text">
+                        <h3>Mult-Post Stories Gain+Glory</h3>
+                        <ul class="ul-items">
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button type="button" class="btn">See Project</button>
+                    </div>
+                </div>
+
+            </article>
+        </section>
+
+        <section class="about">
+
+            <div class="about-text">
+                <h2 class="title">About me</h2>
+                <p class="description">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <button type="button" class="btn">Get My Resume</button>
+            </div>
+
+            <article class="about-container">
+                <div class="about-item">
+                    <img src="./media/icon-languages.png" alt="languages" />
+
+                    <div class="about-item-text">
+                        <h4>Languages</h4>
+                        <ul class="ul-items">
+                            <li>JavaScript</li>
+                            <li>Ruby</li>
+                            <li>html</li>
+                            <li>css</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="about-item">
+                    <img src="./media/icon-frameworks.png" alt="frameworks" />
+
+                    <div class="about-item-text">
+                        <h4>Frameworks</h4>
+                        <ul class="ul-items">
+                            <li>Bootstrap</li>
+                            <li>Ruby on rails</li>
+                            <li>RSpec</li>
+                            <li>Capybara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="about-item">
+                    <img src="./media/icon-skills.png" alt="skills" />
+
+                    <div class="about-item-text">
+                        <h4>Skills</h4>
+                        <ul class="ul-items">
+                            <li>Codekit</li>
+                            <li>GitHub</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                </div>
```

Comment:
Kindly modify this according to the project requirements the technology requirements should be inside of a `ul`, and each of them in a `li`. :recycle: 
![image](https://user-images.githubusercontent.com/98363075/191755681-55cb5605-6fde-4c9c-ab60-cdda9c018c15.png)